scipy/stats/_multicomp.py

```diff
@@ -0,0 +1,311 @@
+        res = minimize_scalar(pvalue_from_stat, method='brent', tol=1e-4)
+        critical_value = res.x
+
+        allowance = critical_value*self._std*np.sqrt(2/len(self.pvalue))
+        return allowance
+
+    def confidence_interval(
+        self, confidence_level: DecimalNumber = 0.95
+    ) -> ConfidenceInterval:
+        """Compute the confidence interval for the specified confidence level.
+
+        The confidence interval corresponds to the mean of the groups
+        +- the allowance.
+
+        Parameters
+        ----------
+        confidence_level : float, optional
+            Confidence level for the computed confidence interval
+            of the estimated proportion. Default is .95.
+
+        Returns
+        -------
+        ci : ``ConfidenceInterval`` object
+            The object has attributes ``low`` and ``high`` that hold the
+            lower and upper bounds of the confidence intervals for each
+            comparison. The high and low values are accessible for each
+            comparison at index ``(i, j)`` between groups ``i`` and ``j``.
+
+        """
+        allowance = self._allowance(confidence_level=confidence_level)
+        diff_means = self._observations_mean - self._control_mean
+        return ConfidenceInterval(
+            low=diff_means-allowance,
+            high=diff_means+allowance
+        )
+
+
+def dunnett(
+    *observations: npt.ArrayLike, control: npt.ArrayLike,
+    alternative: Literal['two-sided', 'less', 'greater'] = "two-sided",
+    random_state: SeedType = None
+) -> DunnettResult:
+    """Dunnett's test.
+
+    Parameters
+    ----------
+    observations1, observations2, ... : 1D array_like
+        The sample measurements for each experiment group.
+    control : 1D array_like
+        The sample measurements for the control group.
+    alternative : {'two-sided', 'less', 'greater'}, optional
+        Defines the alternative hypothesis.
+        The following options are available (default is 'two-sided'):
+
+        * 'two-sided': the means of the distributions underlying the samples
+          are unequal.
+        * 'less': the mean of the distribution underlying the first sample
+          is less than the mean of the distribution underlying the second
+          sample.
+        * 'greater': the mean of the distribution underlying the first
+          sample is greater than the mean of the distribution underlying
+          the second sample.
+    random_state : {None, int, `numpy.random.Generator`}, optional
+        If `random_state` is an int or None, a new `numpy.random.Generator` is
+        created using ``np.random.default_rng(random_state)``.
+        If `random_state` is already a ``Generator`` instance, then the
+        provided instance is used.
+
+    Returns
+    -------
+    res : DunnettResult
+        An object containing attributes:
+
+        statistic : scalar or ndarray
+            The z-score for the test.  For 1-D inputs a scalar is
```

Comment:
```suggestion
            The t-statistic for the test.  For 1-D inputs a scalar is
```
Sometimes a distinction is made and z-score uses population mean and std whereas t statistic is sample mean and std.

And since there are multiple tests when there are multiple samples, really this needs to be more specific about the fact that each element of `statistic` is the statistic between a sample and the control.